server/datastore/mysql/packs.go

```diff
@@ -369,16 +369,20 @@ func (d *Datastore) DeletePack(name string) error {
 
 // Pack fetch fleet.Pack with matching ID
 func (d *Datastore) Pack(pid uint) (*fleet.Pack, error) {
+	return d.packDB(d.reader, pid)
```

Comment:
Could we make this be simply `return packDB(d.reader)`?

Reply:
Good catch, we can indeed! Yeah the idea as you've seen is to do it whenever possible, as in those cases we can't mistakenly use a different DB source.